lib/AST/Module.cpp

```diff
@@ -1163,6 +1163,13 @@ void SourceFile::lookupPrecedenceGroupDirect(
 
 void ModuleDecl::getImportedModules(SmallVectorImpl<ImportedModule> &modules,
                                     ModuleDecl::ImportFilter filter) const {
+  assert((bool)(filter & ImportFilter({
```

Comment:
Is there a method on OptionSet that expresses this more clearly than `&`? Just reads strangely.